bigbluebutton-html5/imports/ui/components/media/service.js

```diff
@@ -8,17 +8,18 @@ import { ACTIONS } from '../layout/enums';
 import UserService from '/imports/ui/components/user-list/service';
 import NotesService from '/imports/ui/components/notes/service';
 
-const LAYOUT_CONFIG = Meteor.settings.public.layout;
 const KURENTO_CONFIG = Meteor.settings.public.kurento;
-const PRESENTATION_CONFIG = Meteor.settings.public.presentation;
 
 const getPresentationInfo = () => {
-  const currentPresentation = Presentations.findOne({
+  const hasCurrent = Presentations.findOne({
     current: true,
   });
 
+  const hasPresentation = Presentations.find({}).count() > 0;
```

Comment:
Let's include `meetingId` as a selector (I see that it wasn't present before)
This should help eliminate rare edge cases where you're moderator in the main room and main room has no presentation but ongoing breakouts do (we're considering doing work on supporting presentations in breakouts, not just a single slide)